src/cli/argument-parser.js

```diff
@@ -32,7 +32,7 @@ const DESCRIPTION = dedent(`
 
     You can use one or more file paths or glob patterns to specify which tests to run.
 
-    More info: http://testcafe.devexpress.com/Documentation/CLI
+    More info: https://devexpress.github.io/testcafe/documentation
```

Comment:
What about more a specific link
https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html ?
Or you think it should be memorable?